docs/resources/etc_hosts.md.erb

```diff
@@ -0,0 +1,62 @@
+---
+title: About the etc_hosts Resource
+---
+
+# etc_hosts
+
+Use the `etc_hosts` InSpec audit resource to test rules set to match IP address's with hostnames.
+## Syntax
+
+An etc/hosts rule specifies an IP address and what it's hostname is along with an aliase names it can have.
+
+## Syntax
```

Comment:
Please see the #2064 PR and apply the same points to the documentation here (remove extra spaces in parens, use the `cmp` matcher to avoid the double-array stuff, removing the return object type from the headers in each of the supported methods/matchers, etc.)